tests/emukit/test_acquisitions.py

```diff
@@ -37,7 +38,8 @@
                      acquisition_test_tuple('multi_source_entropy_search_acquisition', False, np.nan),
                      acquisition_test_tuple('integrated_variance_acquisition', False, np.nan),
                      acquisition_test_tuple('integrated_expected_improvement_acquisition', True, default_grad_tol),
-                     acquisition_test_tuple('integrated_probability_of_improvement_acquisition', False, np.nan)]
+                     acquisition_test_tuple('integrated_probability_of_improvement_acquisition', False, np.nan)
```

Comment:
This missing comma is causing the tests to fail I think:
```suggestion
                     acquisition_test_tuple('integrated_probability_of_improvement_acquisition', False, np.nan),
```

Reply:
many thanks, 
I am very busy on another project at the moment,
I'll have more time to work on the code from tomorrow
cheers
PO